src/screens/Login/login.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { SafeAreaView } from 'react-native';
```

Comment:
```suggestion
import { SafeAreaView } from 'react-native-safe-area-context';
```

Actually it's a good point to re-export `SafeAreaView` in `components/index.ts` to avoid such errors in future